server.js

```diff
@@ -15,12 +15,102 @@ const app = express();
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+const listEndpoints = require('express-list-endpoints');
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json(listEndpoints(app));
 });
 
+const { Schema } = mongoose;
+const thoughtSchema = new Schema({
+  message: {
+    type: String,
+    minlength: 5,
+    maxlength:140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date()
+  }
+});
+
+const Thought = mongoose.model("Thought", thoughtSchema);
+
+app.get("/thoughts", async (req,res)=> {
+  try {  
+  const thought = await Thought.find().sort({createdAt: -1 }).limit(20).exec();
```

Comment:
I am assuming `createdAt: -1`  is the same as `createdAt:  desc` ? Curious what` exec()` does in the code?